src/tucal/plugins/__init__.py

```diff
@@ -2,9 +2,11 @@
 import typing
 
 import tucal.plugins.c187B12
+import tucal.plugins.eventHTU
 
 
 def plugins() -> typing.List[typing.Tuple[str, typing.Type[tucal.Plugin]]]:
     return [
         ('187B12', c187B12.DWI),  # 187.B12 VU Denkweisen der Informatik
+        ('eventHTU', eventHTU.EVENTS),
```

Comment:
`(None, eventHTU.EVENTS)` would be more fitting. The first element of the tuple should only contain the course nr. Please also change the annotation to `...[typing.Tuple[typing.Optional[str], ...]...`.

(And please add a comment like `# https://events.htu.at/`